Chutzpah/TestRunner.cs

```diff
@@ -305,10 +305,15 @@ private IChutzpahWebServerHost SetupWebServerHost(ConcurrentBag<TestContext> tes
             if (contextWithChosenServerConfiguration != null)
             {
                 var webServerConfiguration = contextWithChosenServerConfiguration.TestFileSettings.Server;
-                webServerHost = webServerFactory.CreateServer(webServerConfiguration, ActiveWebServerHost);
 
-                // Stash host object on context for use in url generation
-                contextUsingWebServer.ForEach(x => x.WebServerHost = webServerHost);
+                // create host if it no value (default) or explicit enabled.

```

Comment:
missing a word in the comment: "if it has no value"

Reply:
fixed